README.md

```diff
@@ -208,6 +208,12 @@ Now start the proxy up with the following options:
   -jwt-key="${OAUTH2_PROXY_JWT_KEY}"
 ```
 You can also set all these options with environment variables, for use in cloud/docker environments.
+One tricky thing that you may encounter is that some cloud environments will pass in environment
+variables in a docker env-file, which does not allow multiline variables like a PEM file.
+If you encounter this, then you can create a `jwt_signing_key.pem` file which contains the key
```

Comment:
Done!  Thanks for the feedback.

Reply:
Just in case it's not 100% clear, I did paste in an example of how this is used above:  https://github.com/pusher/oauth2_proxy/pull/111/#issuecomment-475307955